src/core/ext/transport/binder/wire_format/binder_android.cc

```diff
@@ -178,8 +178,10 @@ absl::Status BinderAndroid::PrepareTransaction() {
 
 absl::Status BinderAndroid::Transact(BinderTransportTxCode tx_code) {
   AIBinder* binder = binder_.get();
+  // We only do one-way transaction and thus the output parcle is never used.
+  AParcel* unused_output_parcel;
   return AIBinder_transact(binder, static_cast<transaction_code_t>(tx_code),
-                           &input_parcel_->parcel_, &output_parcel_->parcel_,
+                           &input_parcel_->parcel_, &unused_output_parcel,
```

Comment:
Oops, I think we leak memory here? We will need to call `AParcel_delete` to delete the output parcel

Reply:
How come we didn't notice this leak before?

Reply:
(I mean, it is my fault that in my first prototype I didn't do any memory management and leaves no TODO)

Reply:
You're right. I guess we weren't building Android code with Sanitizers before.

Reply:
Is it even possible to detect leak on Android?

Quick google search shows https://source.android.com/devices/tech/debug/hwasan but I'm not sure how effective it is

Reply:
Done.